setup/setup_vagrant.sh

```diff
@@ -54,7 +54,7 @@ eval "$(rbenv init -)"
 export PATH="$HOME/.rbenv/plugins/ruby-build/bin:$PATH"
 
 # Install ruby
-ruby_version=`cat $REFUGE_PATH/.ruby-version`
+ruby_version=$(cat $REFUGE_PATH/.ruby-version | tr -d '\n\r')
```

Comment:
Kinda neat trick I just learned: `xargs < $REFUGE_PATH/.ruby-version` will do this too